app/inbound/return/save-settlement.js

```diff
@@ -4,8 +4,7 @@ const getCompletedPaymentRequestbyInvoiceNumber = require('../get-completed-paym
 const saveSettlement = async (settlement, transaction) => {
   const matchedPaymentRequest = await getCompletedPaymentRequestbyInvoiceNumber(settlement.invoiceNumber, transaction)
   settlement.paymentRequestId = (matchedPaymentRequest ? matchedPaymentRequest?.paymentRequestId : null) ?? null
-  const savedSettlement = await db.settlement.create(settlement, { transaction })
-  await db.schedule.create({ settlementId: savedSettlement.settlementId }, { transaction })
+  return await db.settlement.create(settlement, { transaction })
```

Comment:
Sonarlint issue: `Redundant use of `await` on a return value.sonarlint(javascript:S4326)`